docs/src/2.4.ManagingCamera.mdx

```diff
@@ -0,0 +1,170 @@
+
+First, let's create a Duck component using [GLTFScene](#/docs/commands/gltfscene) which we'll use to compose with other command components inside Worldview later. Remember to add the actual `Duck.glb` model file in the `src/utils` folder.
+
+```js
+// src/Duck.js
+import React from "react";
+import { GLTFScene } from "regl-worldview";
+import duckModel from "./utils/Duck.glb";
+
+export default function Example() {
+  return (
+    <GLTFScene model={duckModel}>
+      {{
+        pose: {
+          position: { x: 0, y: 0, z: 0 },
+          orientation: { x: 0, y: 0, z: 0, w: 1 },
+        },
+        scale: { x: 3, y: 3, z: 3 },
+      }}
+    </GLTFScene>
+  );
+}
+```
+
+To move the camera, we will need to create a state that gets updated over time. [window.requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) generally matches the display refresh rate. Let's add a [React Hook](https://reactjs.org/docs/hooks-intro.html) that accepts a callback which will be called before each browser repaint.
+
+```js
+// src/utils/useRequestAnimationFrame.js
+import { useEffect, useRef } from "react";
+
+export default function useRequestAnimationFrame(callback) {
+  const callbackRef = useRef();
+  useEffect(() => {
+    callbackRef.current = callback;
+  });
+
+  useEffect(() => {
+    const stop = window.requestAnimationFrame(() => {
+      callbackRef.current();
+    });
+
+    return function cleanup() {
+      cancelAnimationFrame(stop);
+    };
+  });
+}
+```
+
+Lastly, let's hook everything together:
+
+```js
+// src/Example.js
+
+// ...other imports
+
+import Duck from "./Duck";
+import useRequestAnimationFrame from "./utils/useRequestAnimationFrame";
+
+export default function Example() {
+  const cameraMoveSpeed = 1 / 100;
+  // use count to derive camera phi and thetaOffset values
+  const [count, setCount] = useState(0);
+  useRequestAnimationFrame(() => {
+    // update count before each browser repaint
+    const newCount = count + 2 >= steps ? 0 : count + 1;
+    setCount(newCount);
+  });
+
+  // generate cameraState from `count`
+  const cameraState = {
+    ...DEFAULT_CAMERA_STATE,
+    phi: count * cameraMoveSpeed * Math.PI,
+    thetaOffset: count * cameraMoveSpeed * 2 * Math.PI,
+  };
+
+  // ...other existing code
+
+  return (
+    <Worldview cameraState={cameraState}>
+      <Spheres>{[marker]}</Spheres>
+      <Axes />
+      <Duck />
+    </Worldview>
+  );
+}
+```
+
+Now the camera is moving around the scene, giving us 360° view of the duck model!
+
+_Notice we've added `cameraState={cameraState}`, which turns Worldview into a [controlled](https://reactjs.org/docs/forms.html) component. Controlled components usually require an `onChange` handler, but no `onCameraStateChange` prop is needed for our example, since we don't need to know the new internal `cameraState` value._
```

Comment:
Why is just this italicized? Could be more italics throughout tutorial, if there's a reason we are using them.